assets/3dmodelscc0/models/scenes/sniper_rifle.tscn

```diff
@@ -0,0 +1,118 @@
+[gd_scene load_steps=19 format=3 uid="uid://bmjemjgtnpkpo"]
+
+[ext_resource type="PackedScene" uid="uid://c8l60rnugru40" path="res://addons/godot-xr-tools/objects/pickable.tscn" id="1_lpv7q"]
+[ext_resource type="PackedScene" uid="uid://c25yxb0vt53vc" path="res://addons/godot-xr-tools/objects/grab_points/grab_point_hand_left.tscn" id="2_x4cgt"]
+[ext_resource type="Animation" uid="uid://ddbo6ioa282en" path="res://addons/godot-xr-tools/hands/animations/left/Pistol.res" id="3_np0oa"]
+[ext_resource type="Script" path="res://addons/godot-xr-tools/hands/poses/hand_pose_settings.gd" id="4_mworp"]
+[ext_resource type="PackedScene" uid="uid://ctw7nbntd5pcj" path="res://addons/godot-xr-tools/objects/grab_points/grab_point_hand_right.tscn" id="5_sjb82"]
+[ext_resource type="PackedScene" uid="uid://da2qgxxwwitl6" path="res://addons/godot-xr-tools/objects/highlight/highlight_ring.tscn" id="6_x8gva"]
+[ext_resource type="Animation" uid="uid://csp3fdknowmi5" path="res://addons/godot-xr-tools/hands/animations/right/Pistol.res" id="7_ll2st"]
+[ext_resource type="PackedScene" uid="uid://jg1pijfn8ea3" path="res://assets/3dmodelscc0/models/sniper_rifle/sniper_rifle.glb" id="7_tk4wb"]
+[ext_resource type="Material" uid="uid://pcfey2xeiuoe" path="res://assets/3dmodelscc0/models/sniper_rifle/sniper_rifle.material" id="8_7v2rw"]
+[ext_resource type="Script" path="res://assets/3dmodelscc0/models/sniper_rifle/firearm_trigger.gd" id="9_0ljhu"]
+[ext_resource type="Animation" uid="uid://dlxa6f6hwurka" path="res://addons/godot-xr-tools/hands/animations/left/Default pose.res" id="11_bhts3"]
+[ext_resource type="Animation" uid="uid://ky28birj4su6" path="res://addons/godot-xr-tools/hands/animations/right/Default pose.res" id="13_46eib"]
+[ext_resource type="PackedScene" uid="uid://ce7vysyvondf8" path="res://addons/godot-xr-tools/objects/snap_zone.tscn" id="15_coy4c"]
+
+[sub_resource type="BoxShape3D" id="BoxShape3D_apd71"]
+size = Vector3(0.0449182, 0.225, 1.145)
+
+[sub_resource type="Resource" id="Resource_ok7hp"]
+script = ExtResource("4_mworp")
+open_pose = ExtResource("11_bhts3")
+closed_pose = ExtResource("3_np0oa")
+
+[sub_resource type="Resource" id="Resource_kscym"]
+script = ExtResource("4_mworp")
+open_pose = ExtResource("13_46eib")
+closed_pose = ExtResource("7_ll2st")
+
+[sub_resource type="Resource" id="Resource_tbrsh"]
+script = ExtResource("4_mworp")
+open_pose = ExtResource("3_np0oa")
+closed_pose = ExtResource("3_np0oa")
+
+[sub_resource type="Resource" id="Resource_6toxn"]
+script = ExtResource("4_mworp")
+open_pose = ExtResource("7_ll2st")
+closed_pose = ExtResource("7_ll2st")
+
+[node name="SniperRifle" instance=ExtResource("1_lpv7q")]
+ranged_grab_method = 2
+second_hand_grab = 3
```

Comment:
There doesn't seem to be an  enum for `second_hand_grab = 3`, is this a left over from testing?

Reply:
Yes - there was an earlier attempt at simplifying the two handed grab approach into "grab" vs "aim" but the weighting system just provided better flexibility. This hold-over has now been fixed.